README.md

```diff
@@ -26,6 +26,13 @@ let driver = new WindowsDriver();
 await driver.createSession(defaultCaps);
 ```
 
+## Additional WindowsDriver only capabilities
+
+|Capability|Description|Values|
+|----------|-----------|------|
+|`createSessionTimeout`|Timeout in milliseconds used to retry `WinAppDriver` start session creation. This is workaround for a long startup duration UWP application error: `Original error: Failed to locate opened application window with appId: TestCompany.my_app4!App, and processId: 8480`. Default value `20000`|e.g., `15000`|
```

Comment:
This is the workaround for the long startup times of UWP applications (aka `Failed to locate opened application window with appId: TestCompany.my_app4!App, and processId: 8480`)